packages/block-library/src/template-part/index.js

```diff
@@ -20,6 +20,6 @@ export { metadata, name };
 export const settings = {
 	title: _x( 'Template Part', 'block title' ),
 	keywords: [ __( 'template part' ) ],
-	__experimentalLabel: ( { slug } ) => startCase( slug ),
+	__experimentalLabel: ( slug ) => startCase( slug ),
```

Comment:
This change is wrong because this function is called with block's `attributes`.

Reply:
wrong commit remnant, was seeing if `{ name: slug }` would have any effect in bypassing the `:`